code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm

```diff
@@ -120,6 +120,39 @@
 		udder.reagents.clear_reagents()
 		regenerate_icons()
 
+/mob/living/simple_animal/hostile/asteroid/gutlunch/grublunch
+	name = "grublunch"
+	wanted_objects = list() //They don't eat.
```

Comment:
may want to comment here that if someone changes this they need to consider the fact babies could be made from babies, although this is a MINOR issue (sorry had to)

Reply:
Babies can't make babies because they are neuter. So even if someone changes it, it won't matter.